flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/PredefinedOptions.java

```diff
@@ -60,9 +59,7 @@
      *   <li>setInfoLogLevel(InfoLogLevel.HEADER_LEVEL)
```

Comment:
We should also remove such comments.
The same for all other options defined in this file.